.travis/install

```diff
@@ -3,10 +3,14 @@
 echo "create database lobby_db" | psql -h localhost -U postgres -d postgres
 echo "create user lobby_user with password 'postgres'" | psql -h localhost -U postgres -d postgres
 echo "alter database lobby_db owner to lobby_user" | psql -h localhost -U postgres -d postgres
-sed -i "s/.0$/.$TRAVIS_BUILD_NUMBER/" game-core/src/main/resources/META-INF/triplea/product.properties
+
+if [ ! -z "$TRAVIS_BUILD_NUMBER" ]; then
+  sed -i "s/.0$/.$TRAVIS_BUILD_NUMBER/" game-core/src/main/resources/META-INF/triplea/product.properties
+fi
+
 RELEASE_VERSION=$(sed 's/version\s*=\s*//' game-core/src/main/resources/META-INF/triplea/product.properties)
 ./gradlew flywayMigrate
-./gradlew shadowJar
+./gradlew -b http-server/build.gradle shadowJar
```

Comment:
I believe you can just use `./gradlew :http-server:shadowJar` here, no need to manually specify the gradle build file